Cookbook/Technical-Documents/FastlaneMatch.md

```diff
@@ -208,4 +212,6 @@ Fastlane match requires that provisioning profiles are named AdHoc_`bundle ident
 
 ## Problems and trouble-shooting
 
+- When you run `fastlane match` first time it will ask you to enter the password from developers portal. This password is stored in iOS team's 1Password vault. If you used the wrong password you will not be asked to change it. You then need to go to your keychain, find the entry for this password (it will be named after the email of the account prefixed with `deliver.`) and delete it. After that next time you run `faslane match` it will ask you for the password again.
```

Comment:
```suggestion
- When you run `fastlane match` first time it will ask you to enter the password from Apple developer center. This password is stored in iOS team's 1Password vault. If you used the wrong password you will not be asked to change it. You then need to go to your keychain, find the entry for this password (it will be named after the email of the account prefixed with `deliver.`) and delete it. After that next time you run `faslane match` it will ask you for the password again.
```